src/pip/_internal/commands/install.py

```diff
@@ -1,9 +1,11 @@
+import ctypes
```

Comment:
`ctypes` is an optional module (only available in CPython when compiled with libffi), so this needs a fallback. I think it’s reasonable to simply disable the superuser check on Windows if ctypes is unavailable.

Reply:
thanks for the flag. we can catch the import with the existing try-except:

```
             if sys.platform == 'darwin' or sys.platform == 'linux':
                 is_sudo = os.getuid() == 0
             elif sys.platform == 'win32' or sys.platform == 'cygwin':
+                import ctypes
                 is_sudo = ctypes.windll.shell32.IsUserAnAdmin() != 0
-        except AttributeError:
+        except (ImportError, AttributeError):
             is_sudo = False
```